Tasks/Common/VstsAzureHelpers_/InitializeFunctions.ps1

```diff
@@ -63,8 +63,12 @@ function Initialize-AzureSubscription {
     if ($Endpoint.Auth.Scheme -eq 'ServicePrincipal' -and !$script:azureModule -and (Get-Command -Name "Clear-AzureRmContext" -ErrorAction "SilentlyContinue")) {
         Write-Host "##[command]Clear-AzureRmContext -Scope Process"
         $null = Clear-AzureRmContext -Scope Process
-        Write-Host "##[command]Clear-AzureRmContext -Scope CurrentUser -Force -ErrorAction SilentlyContinue"
-        $null = Clear-AzureRmContext -Scope CurrentUser -Force -ErrorAction SilentlyContinue
+    }
```

Comment:
Area you planning to port #9327 separately? Can we get all required changes in single Ask mode?

Reply:
this file is not present in M143.